test/rsa_mp_test_msg

```diff
@@ -0,0 +1 @@
+The time of life is short; to spend that shortness basely, it would be too long.(A Midsummer Night’s Dream 1.1)
```

Comment:
It was asserted that it's more convenient to have it as separate file. I really fail to see why. It can as well be inlined into corresponding recipe.

Reply:
Data and code should be separated. In case of changing the text into a larger one in the future, the code stays untouched.

Reply:
> Data and code should be separated.

It's not universal statement. For example, if it was, then internal tests would look very different :-) My concern is that presence of this file will evoke [unnecessary] question where does it belong after a while. I mean memories fade, you know, and some will look at it for the first time. If feeling that it has to be separate file that strong, then it probably should be moved to test/recipies/15-test_mp_rsa_data...

Reply:
Ahh, another advantage of this might be: this file could be reused by other test cases as a generic plain text source. So the name of this might be changed to something like plain_raw_material and more sentences should put into this file...

Reply:
Hmmm, one idea that will probably have everyone shut up is to use the recipe itself as cleartext input file ;-)

Have a look in `test/recipes/20-test_enc.t` for an example on how to do it.

Reply:
Nah, the receipt file will be too long for an RSA key to do the crypto-operations...

Reply:
D'oh...  ok, silly me.

Reply:
Unless get the self-recipe-file hash'ed after reading it... ;-)